### PR TITLE
fixed local prefix setting

### PR DESCRIPTION
**Pull request by jung1411**
# Befor putting PR (put x in bracket if you ran it.);

- [x] Run `pylint $(git ls-files '*.py')` passes

# Overview

Describe your pull request. Attach github issue link

https://github.com/dhmoon91/discord/issues/3

Description: Made .env variable to store the prefix to run locally. To use in local, just add `LOCAL_PREFIX = "!"` or anything within the double quotes in .env file.

### CODE REVIEW

**Review by dhmoon91 — APPROVED** [review]

**Comment by dhmoon91** [comment]
update readme.md

File: bot.py [hunk 1]
```diff
@@ -21,9 +21,12 @@
 
 load_dotenv()
 TOKEN = os.getenv("DISCORD_TOKEN")
+LOCAL_PREFIX = os.getenv("LOCAL_PREFIX")
```
Comment (dhmoon91) [inline bot.py]: change name to `LOCAL_BOT_PREFIX`

**Comment by dhmoon91** [comment]
Left one comment. 
Feel free to merge it after updating readme and addressing my comment

File: bot.py (hunk 1 above)
Comment (jung1411) [reply bot.py]: changed

**Pull request merged**